apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -166,9 +166,25 @@
       "expectedTestAppValue": "Receieved error {\"errorCode\":1000,\"message\":\"Encountered an error\"}"
     },
     {
-      "title": "shareAppContentToStage API Call (<2.19.0) - Success",
+      "title": "shareAppContentToStage API Call (host SDK <2.8.0) - Success",
```

Comment:
Not really sure you need to update the title. The `hostSdkVersion.web` value is already prepended in the test title by the framework. This is how it appears in the logs:

"Target Web Host SDK v-<2.8.0 - shareAppContentToStage API Call (host SDK <2.8.0) - Success"

Reply:
I was updating the title for the benefit of people reading the JSON and not really because of the logging output (though obviously it could help there too, but not in this case)

I'm just trying to fit into whatever the existing system is here: should I remove the version from the title?

Reply:
I think it's helpful to folks reading the code, so I'm fine with leaving it in. It doesn't exactly match convention (if one previous change can be considered "convention"), but I think it's an improvement.